samples/matter/weather_station/src/app_task.cpp

```diff
@@ -81,6 +78,9 @@ int AppTask::Init()
 	PlatformMgr().AddEventHandler(AppTask::ChipEventHandler, 0);
 #endif
 
+	/* Automatically enable BLE advertisment */
```

Comment:
Can we use `CHIP_ENABLE_PAIRING_AUTOSTART`? And start NFC Tag in `ChipEventHandler`?

Reply:
Done